components/header/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Heading,
+  Box,
+  Link,
+  Flex,
+  Spacer,
+  Button,
+  Stack,
+  useColorMode
+} from '@chakra-ui/react'
+import { default as NextLink } from 'next/link'
+import { useRouter } from 'next/router'
+import useTranslation from 'next-translate/useTranslation'
+import { MoonIcon, SunIcon } from '@chakra-ui/icons'
+import setLanguage from 'next-translate/setLanguage'
+
+const Header: React.FC = () => {
+  const router = useRouter()
+  const { colorMode, toggleColorMode } = useColorMode()
+  const { t, lang } = useTranslation('common')
+  return (
+    <Box p="4">
+      <Flex>
+        <Box p={2}>
+          <NextLink passHref href="/" locale={router.locale}>
+            <Link href="/">
+              <Heading size="md">{t('LAYOUT_HEADING')}</Heading>
+            </Link>
+          </NextLink>
+        </Box>
+        <Spacer />
+        <Stack direction="row" spacing={4}>
+          <NextLink passHref href="/" locale={router.locale}>
+            <Link _focus={{ boxShadow: 'none' }} href="/" p={4}>
+              {t('HOME_LINK')}
+            </Link>
+          </NextLink>
+          <NextLink passHref href="/register" locale={router.locale}>
+            <Link _focus={{ boxShadow: 'none' }} p={4}>
+              {t('REGISTER_LINK')}
+            </Link>
+          </NextLink>
+          <NextLink passHref href="/nengajo" locale={router.locale}>
```

Comment:
It's nice to see how the links will look, but the pages do not exist yet. Could you comment the /nengajo and /collection links?

Reply:
Sure.
I will change this.

Reply:
Thank you!